stdlib/asyncio/events.pyi

```diff
@@ -67,33 +68,37 @@ class _TaskFactory(Protocol):
         self, __loop: AbstractEventLoop, __factory: Coroutine[Any, Any, _T] | Generator[Any, None, _T]
     ) -> Future[_T]: ...
 
-class Handle:
+class Handle(Generic[Unpack[_Ts]]):
     _cancelled: bool
-    _args: Sequence[Any]
+    _args: tuple[Unpack[_Ts]]
     def __init__(
-        self, callback: Callable[..., object], args: Sequence[Any], loop: AbstractEventLoop, context: Context | None = None
+        self,
+        callback: Callable[[Unpack[_Ts]], object],
+        args: tuple[Unpack[_Ts]],
+        loop: AbstractEventLoop,
+        context: Context | None = None,
     ) -> None: ...
     def cancel(self) -> None: ...
     def _run(self) -> None: ...
     def cancelled(self) -> bool: ...
     if sys.version_info >= (3, 12):
         def get_context(self) -> Context: ...
 
-class TimerHandle(Handle):
+class TimerHandle(Handle[Unpack[_Ts]]):
     def __init__(
         self,
         when: float,
-        callback: Callable[..., object],
-        args: Sequence[Any],
+        callback: Callable[[Unpack[_Ts]], object],
+        args: tuple[Unpack[_Ts]],
         loop: AbstractEventLoop,
         context: Context | None = None,
     ) -> None: ...
     def __hash__(self) -> int: ...
     def when(self) -> float: ...
-    def __lt__(self, other: TimerHandle) -> bool: ...
-    def __le__(self, other: TimerHandle) -> bool: ...
-    def __gt__(self, other: TimerHandle) -> bool: ...
-    def __ge__(self, other: TimerHandle) -> bool: ...
+    def __lt__(self, other: TimerHandle[Unpack[tuple[Any, ...]]]) -> bool: ...
+    def __le__(self, other: TimerHandle[Unpack[tuple[Any, ...]]]) -> bool: ...
+    def __gt__(self, other: TimerHandle[Unpack[tuple[Any, ...]]]) -> bool: ...
+    def __ge__(self, other: TimerHandle[Unpack[tuple[Any, ...]]]) -> bool: ...
```

Comment:
not exactly the prettiest type hints in the world, but I think this is the correct way of doing this...